actions/platform/chrome.py

```diff
@@ -0,0 +1,99 @@
+import json
+
+from actions.slot_match import FuzzySlotMatchOption
+from common.requests import send_console_request
+
+unsure_option = FuzzySlotMatchOption(
+    {"href": "unsure", "title": "unsure", "group": "unsure"},
+    [
+        "not sure",
+        "unsure",
+        "idk",
+        "I'm not sure",
+        "no clue",
+        "I have no idea",
+        "other",
+        "I don't know the name of the service",
+        "I don't know",
+        "other",
+    ],
+)
+
+
+async def get_user(tracker):
+    # struggles to be a json response, manually doing it here
+    resp = await send_console_request(
+        "chrome-service",
+        "/api/chrome-service/v1/user",
+        tracker,
+        "get",
+    )
+
+    response, content = resp
+
+    return response, json.loads(content)
+
+
+async def create_service_options(tracker) -> list[FuzzySlotMatchOption]:
+    response, content = await get_generated_services(tracker)
+    if response.ok:
+        return parse_generated_services(content)
+    else:
+        # failed to reach the chrome service
+        return [unsure_option]
+
+
+def parse_generated_services(content):
+    options = [unsure_option]
+    for category in content:
+        for link in category["links"]:
+            group_title = link.get("title")
+            if "isGroup" in link and link["isGroup"]:
+                for sublink in link["links"]:
+                    if "isExternal" in sublink and sublink["isExternal"]:
+                        # not really a service
+                        continue
+                    synonyms = []
+                    value = {"group": group_title}
```

Comment:
Could all the `metadata` (group, app_id, etc) be returned as a separate dictionary?
To keep consistent the `FuzzySlotMatchOption` or alternatively put the metadata in the FuzzySlotMatchOption as a different param.

Reply:
For this one, I could return everything as an array of dicts from the parse_generated_service function, then convert to the FuzzySlotMatchOptions in the create_service_options function. Would that cover it?

Reply:
that sounds good, thanks!